server/controllers/file.controller.js

```diff
@@ -39,6 +40,10 @@ function getAllDescendantIds(files, nodeId) {
 
 function deleteMany(files, ids) {
   ids.forEach(id => {
+    // Delete file from S3 bucket
+    if (!(files.id(id).url === undefined)) {
+      deleteObjectFromS3(files.id(id).url);
+    }
     files.id(id).remove();
```

Comment:
should this be within the `deleteObjectFromS3` callback? will this cause an issue where the API returns but the files are still be deleted from S3 and then they don't get deleted properly?